preferences/linux-bios/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../base
+
+components:
+  - ./metadata
+  - ../components/diskbus-virtio-blk
+  - ../components/interfacemodel-virtio-net
+  - ../components/rng
+  - ../components/bios
+
+patches:
+  - target:
+      kind: VirtualMachinePreference
+    patch: |-
+      - op: replace
+        path: /metadata/name
+        value: linux-bios
```

Comment:
Should we just call this `linux`?

Reply:
Sounds good, but it should still make use of `preferredUseBios`? Or is it `preferredUseBios` the default option if none is provided?

Reply:
Done.